commands/auth.go

```diff
@@ -342,7 +342,7 @@ func renewCmdF(cmd *cobra.Command, args []string) error {
 		credentials.AuthToken = c.AuthToken
 
 	default:
-		return errors.New(fmt.Sprintf("Invalid auth method \"%s\"", credentials.AuthMethod))
+		return errors.Errorf("invalid auth method %q", credentials.AuthMethod)
```

Comment:
Would you mind using `errors.WithMessagef` instead?

Reply:
I think we can't since there is no error to add new message.

From the `errors.WithMessagef` doc: 
> WithMessagef annotates err with the format specifier. If err is nil, WithMessagef returns nil.

Reply:
NVM, by mad